File: python/8.11/example4.py
from random import randint # Importing randint from random

def totalValues(): # Creating the definiton for totalValues
    arrayData = [] # Setting the value to []
    for y in range(1, 51): # Running a loop 50 times
        arrayData += [randint(0, 25565)] # Adding a random number to arrayData
    total = 0 # Setting the value to 0
    for x in range(0, 50): # Running a loop 50 times
        total = total + arrayData[x] # adding the value of the current point in the array to the value
    return total # Reterning total

print(str(totalValues())) # Printing

File: java/8.11/example4.java
import java.util.Random;

public class example4 {
    public static void main(String[] args) {
        System.out.println(totalValues());
    }

    private static Integer totalValues() {
        Random rand = new Random();
        Integer[] arrayData = new Integer[50];
        for (Integer y = 0; y < 50; y++) {
            arrayData[y] = rand.nextInt(25565);
        } 
        Integer total = 0;
        for (Integer x = 0; x < 50; x++) {
            total = total + arrayData[x];
        }
        return total;
    }
}
